src/vulkan/descriptor.cc

```diff
@@ -53,18 +53,45 @@ Descriptor::Descriptor(DescriptorType type,
                        VkDevice device,
                        uint32_t desc_set,
                        uint32_t binding)
-    : device_(device),
-      descriptor_set_(desc_set),
+    : descriptor_set_(desc_set),
       binding_(binding),
-      type_(type) {}
+      type_(type),
+      device_(device) {}
```

Comment:
Why did this change around?

Reply:
Because I just moved `device_` to `private:`, the order of members became different.
Without this change, there is an initializing order error from `-Werror=reorder`.